src/http/client.rs

```diff
@@ -442,6 +451,18 @@ impl EspHttpConnection {
             panic!("connection is not in response phase");
         }
     }
+
```

Comment:
I think you don't need this method at all. [This post](https://www.esp32.com/viewtopic.php?t=17314) says, that as long as you pass `-1` to `esp_http_clienbt_open`, the client would assume chunked encoding.